Improve rosetta docs (#727)
o1-labs/docs2: docs/exchange-operators/rosetta/samples/track-deposits.mdx

---
title: Tracking deposits
description: Learn how to track deposits on the given address
---

> Once we have everything ready, let's implement deposit tracking.
> 
> For that, we will iterate through all transactions in each block to search for `payment_receiver_inc` operation, with an address matching the address which we are interested in:
> 

Reviewer: ```suggestion
After everything is ready, you can implement deposit tracking.

For that, iterate through all transactions in each block to search for `payment_receiver_inc` operation with an address matching the address in which you are interested:

```
